packages/verification-pool-api/package.json

```diff
@@ -12,6 +12,7 @@
     "set-config": "ts-node ./deployment/set-config.ts --celo-env",
     "set-env": "ts-node ./deployment/set-env.ts --celo-env",
     "deploy": "ts-node ./deployment/deploy.ts --celo-env",
+    "deploy:install": "./deploy.sh",
```

Comment:
Why do we need this new job and the old deploy job? Can we merge the two?

Reply:
The old deploy job does not run `yarn` and install the dependencies. celotool uses the old deploy job but it is used on a monorepo where `yarn` is already run.
Hence a new job.